velox/exec/PlanNodeStats.cpp

```diff
@@ -100,6 +103,9 @@ std::string PlanNodeStats::toString(bool includeInputStats) const {
     out << ", Physical written output: " << succinctBytes(physicalWrittenBytes);
   }
   out << ", Cpu time: " << succinctNanos(cpuWallTiming.cpuNanos)
+      << ", Add input cpu time: " << succinctNanos(addInputTiming.cpuNanos)
```

Comment:
Maybe shorten a bit more readability and move to the end of the message

CPU time: ..., Blocked all time: ..., CPU breakdown: I / O / F (addInput / getOutput / finish)